src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -321,6 +321,10 @@ sap.ui.define([
 			aResourcePath = [oResource.resourceType , "§" + oResource.expansion.identifier + "§"];
 			oResource = oResource.expansion.contains;
 		} else {
+			// generate a uuid id incase resource id is not present in the response
```

Comment:
[style] "incase" add a missing whitespace between "in" and "case"

Reply:
done